client/src/modules/stock/exit/exit.js

```diff
@@ -285,6 +285,13 @@ function StockExitController(
       .then(inventories => {
         vm.loading = false;
         vm.selectableInventories = angular.copy(inventories);
+
+        // map of inventories by inventory uuid
+        vm.mapSelectableInventories = _.reduce(vm.selectableInventories, (aggregate, value) => {
```

Comment:
I'm not sure it's worth adding the entire lodash library just for a _.reduce().  Isn't `vm.selectedInventories` an array?

Reply:
Also, you might consider making this a `new Store()`.  I think it's probably what you want - an ability to look up the entire inventory by its `uuid`.  Like this:
```js
let selectableInventoriesStore = new Store({ key : 'inventory_uuid', data : vm.selectableInventories });
```
and then later you can use it like:
```js
patient.invoice.items.forEach((item) => {
  const inventory = selectableInventoriesStore.get(item.inventory_uuid);
  if (inventory) {
    addItems(1);
    // do something
  }
});